src/navigation/RootStackNavigator.types.ts

```diff
@@ -1,10 +1,12 @@
-import { Accommodation } from 'src/types';
+import { StackScreenProps } from '@react-navigation/stack';
+import { SignUpParams, Accommodation } from 'src/types';
+
 
 export type RootStackParamList = {
   Main: undefined;
   Signup?: undefined;
   Signin?: undefined;
-  VerifyEmail?: undefined;
+  VerifyEmail: { data: SignUpParams };
```

Comment:
`data` is a very generic name, please, be more accurate and specific with naming